google/cloud/stream_range.h

```diff
@@ -165,9 +157,9 @@ class StreamRange {
   StreamRange(StreamRange const&) = delete;
   StreamRange& operator=(StreamRange const&) = delete;
   // NOLINTNEXTLINE(performance-noexcept-move-constructor)
-  StreamRange(StreamRange&&) noexcept(IsMoveNoexcept<T>()) = default;
+  StreamRange(StreamRange&&) = default;
```

Comment:
nit: maybe you can move the `// NOLINTNEXTLINE(...)` to `// NOLINT` ... on the next line. Ha! that rhymes.

Reply:
I tried but it's still too long and will wrap.